.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     docker:
       # specify the version
       - image: colinmerkel/mushu-ci:latest
-      
+
```

Comment:
Why change formatting?

Reply:
Atom automatically does that on save, sorry.